### DISCUSSION TITLE
Google is indicating that we have some issue connecting to payment

### DISCUSSION THREAD

**Discussion by Abdulahaditsol** [hyochan/react-native-iap#2185] [Show and tell]
Hi everyone I am facing issue while testing IAP in android.
after run finishTransaction it shows Google is indicating that we have some issue connecting to payment.
I followed migration guide but nothing is working.
react-native version  "react-native": "0.63.3",
RNIAP version   "react-native-iap": "^11.0.7",
I shall be thankful if anyone suggests to me where I am going wrong.
Thanks

**Comment by oajh04** [comment]
Did you solve this problem?

**Comment by PatriciaSauer** [comment]
I think I'm experiencing the same issue. FinishTransaction goes through and afterwards an error is thrown. Could you solve this issue?
